shared/src/types/ethereum_events/vote_extensions.rs

```diff
@@ -252,16 +260,16 @@ mod tests {
 
         // assume both v1 and v2 saw the same events,
         // so each of them signs `ext` with their respective sk
-        let ext_1 = Signed::new(&sk_1, ext.clone());
-        let ext_2 = Signed::new(&sk_2, ext);
+        let ext_1 = Signed::new(&sk_1, ext(validator_1.clone()));
+        let ext_2 = Signed::new(&sk_2, ext(validator_2.clone()));
 
         let ext = vec![ext_1, ext_2];
 
         // we have the `Signed<VoteExtension>` instances we need,
         // let us now compress them into a single `VoteExtensionDigest`
         let signatures: Vec<(_, Address)> = vec![
```

Comment:
We can probably remove the type signature (`Vec<(_, Address)>`). This was only here because the compiler was complaining about not being able to resolve the type of `.into()`.

Reply:
:+1: